contracts/GSN/GSNRecipient.sol

```diff
@@ -3,18 +3,23 @@ pragma solidity ^0.5.0;
 import "./IRelayRecipient.sol";
 import "./IRelayHub.sol";
 import "./Context.sol";
-import "./bouncers/GSNBouncerBase.sol";
 
 /**
  * @dev Base GSN recipient contract: includes the {IRelayRecipient} interface and enables GSN support on all contracts
```

Comment:
I would add a paragraph explaining that this contract is abstract and listing the abstract functions.